apps/api/src/app/providers/storage/interfaces/file.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface File {
+  path: string;
```

Comment:
Change this to return the `fileName`, not the `path` to the file. Path isn't relevant when the data is returned.